src/main/java/L7Z2/ToUpperCase.java

```diff
@@ -0,0 +1,12 @@
+package L7Z2;
+
+import java.util.Objects;
+
+public class ToUpperCase {
+    public static String convertToUpperCase(String text) {
+        if (Objects.isNull(text) || text.equals("")) {
+            throw new NullPointerException();
```

Comment:
Nigdy nie rzuca się NullPointerException - ty powinieneś tak pisać program, aby nigdy tego błędu nie dostać, a co dopiero sam celowo go produkować ;) Jak już to lepiej zwrócić tutaj pusty string, albo jakiś "IllegalArgumentException", albo, skoro już chcesz NullPointerException to po prostu:

```
return text.toUpperCase();
```
bo wynik będzie dokładnie ten sam - jeżeli text == null to zwróci NullPointerException :P